sdk/purview/Azure.Analytics.Purview.Scanning/src/Generated/PurviewClassificationRuleClient.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<doc>
+  <assembly>
+    <name>Azure.Analytics.Purview.Scanning</name>
+  </assembly>
+  <members>
+    <member name="M:Azure.Analytics.Purview.Scanning.PurviewClassificationRuleClient.GetPropertiesAsync(Azure.RequestContext)">
+      <remarks>
+            Schema for <c>Response Body</c>:
+        <code>{
+              id: string,
+              name: string,
+              kind: "System" | "Custom"
+            }
+        </code>
+            Schema for <c>Response Error</c>:
+        <code>{
+              error: {
+                code: string,
+                message: string,
+                target: string,
+                details: [
+                  {
+                    code: string,
+                    message: string,
+                    target: string,
+                    details: [ErrorModel]
+                  }
+                ]
+              }
+            }
+        </code>
+
+      </remarks>
+    </member>
+  </members>
+</doc>
```

Comment:
Can we move this XML file under `Generated/Docs` folder? so that we can easily look for the source files under the Generated folder and XML files under `Generated/Docs` folder?

Reply:
agree. previously in the meeting, when Crystal was talking about the proper location of those XML files, I thought maybe int he further a proper folder structure of `Generated` could be `Generated/Src`, `Generated/Docs`, `Generated/Test` etc.